readthedocs/core/static-src/core/js/doc-embed/version-compare.js

```diff
@@ -25,11 +25,14 @@ function init(data) {
       .attr('href', currentURL)
       .text(data.slug);
 
-    var body = $("div.body");
-    if (!body.length) {
-        body = $("div.document");
+    var selectors = ['[role=main]', 'main', 'div.body', 'div.document'];
+    for (var i = 0; i < selectors.length; i += 1) {
+        var body = $(selectors[i]);
```

Comment:
Were we ok using `let` instead of `var`? Or there was some browser compatibility we were worried about?

Reply:
FWIW, if we're using let, we should also use it (and const) as much as possible.

Reply:
We should keep it the same as the existing code, to keep things consistent.